.meta/sinks/statsd.toml.erb

```diff
@@ -1,41 +0,0 @@
-[sinks.statsd]
-title = "Statsd"
-noun = "Statsd"
-beta = false
-common = false
-delivery_guarantee = "best_effort"
-<%= render("_partials/descriptions/_statsd.toml") %>
-egress_method = "streaming"
-features = [
-  "Stream metrics over a StatsD UDP protocol.",
-  "Automatically aggregate metrics at the edge for improved performance.",
-]
-function_category = "transmit"
-healthcheck = false
-requirements = {}
-input_types = ["metric"]
-write_to_description = "[StatsD][urls.statsd] metrics service"
-
-<%= render("_partials/fields/_component_options.toml", type: "sink", name: "statsd") %>
-
-[sinks.statsd.options.address]
-type = "string"
-common = true
-examples = ["127.0.0.1:8125"]
-default = "127.0.0.1:8125"
-description = "The UDP socket address to send stats to."
-
-[sinks.statsd.options.namespace]
-type = "string"
-common = true
-examples = ["service"]
-required = false
-description = "A prefix that will be added to all metric names."
-
-[[sinks.statsd.examples]]
```

Comment:
We don't use examples anymore?

Reply:
Yeah, this is what I need help with as described in #4303 😄 .